### ISSUE TITLE
Invalid password with the correct password.

### ISSUE THREAD

**Issue by lwatsondev** [andykais/bitwarden-dmenu#4]
```
rev@serenity ~> bitwarden-dmenu --on-error 'xargs notify-send --urgency=low' --verbose
$ dmenu -p Password: -nf black -nb black 

$ /home/rev/.nvm/versions/node/v8.12.0/lib/node_modules/bitwarden-dmenu/node_modules/.bin/bw unlock $'******' --raw 

Error: Invalid master password.
    at module.exports.args (/home/rev/.nvm/versions/node/v8.12.0/lib/node_modules/bitwarden-dmenu/src/exec-bitwarden-cli.js:13:11)
    at getSessionVar (/home/rev/.nvm/versions/node/v8.12.0/lib/node_modules/bitwarden-dmenu/src/index.js:32:21)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
begin cleanup 

/tmp/bitwarden-session.txt already removed. 

$ /home/rev/.nvm/versions/node/v8.12.0/lib/node_modules/bitwarden-dmenu/node_modules/.bin/bw lock 

bitwarden is locked.
clipboard is cleared.
```

**Comment by andykais** [comment]
Alright, figured it out. So this was due to a fix I made for a previous issue (#2) surrounding single quotes in passwords. There are some other characters that were treated weirdly after this fix that I failed to realize. Now, single quote strings in passwords are the only things escaped (and like so `pw'quote` becomes `'pw'\''quote'`). The fix is applied in this commit https://github.com/andykais/bitwarden-dmenu/commit/8156aee0cf9c31f090149ce8de2152470d51fda2 and `bitwarden@1.2.2`

**Comment by andykais** [comment]
update: escaping strings is a pain, and I have decided to put the responsibility onto `child_process.spawn` where it should be. Everything passed to `bw` now is an arg for `spawn` which is escaped (in a similar manner to what `v1.2.2` was doing.

New fix is in https://github.com/andykais/bitwarden-dmenu/commit/cc8cfaa12602e099fcf295908753be7078c97a9f and `bitwarden@1.2.3`. This performs the exact same as `v1.2.2` but this is safer for future bugs that may be lurking about.
